runtime/port/win32/j9shsem.c

```diff
@@ -60,6 +62,33 @@ j9shsem_params_init(struct J9PortLibrary *portLibrary, struct J9PortShSemParamet
  	params->proj_id = 1; /* parameter used with semName to generate semaphore key */
  	return 0;
 }
+
+/*
+ * Convert a UTF-8 string to UTF-16 and prepend a global name space prefix.
+ * This allows a user with multiple sessions to share the semaphore.
+ * @param portLibrary port library
+ * @param source original UTF-8 string
+ * @param destination buffer to receive the UTF-16 string
+ * @param destinationSize size of destination buffer in UTF-16 characters
+ * @return the destination buffer or NULL on failure.
+ */
+static wchar_t*
+convertToWindowsFormat(OMRPortLibrary *portLibrary, const char *source, wchar_t *destination, uintptr_t destinationSize) {
+	wchar_t *startPosition = destination;
```

Comment:
Please use a better name than `startPosition`

Reply:
Okay.  Changed to ```destinationCursor```.